test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -311,6 +324,16 @@ void verifyTables(HashMap<String, CorfuTable<Long, Long>> tables0, HashMap<Strin
             }
     }
 
+    void waitData(HashMap<String, CorfuTable<Long, Long>> tables, HashMap<String, HashMap<Long, Long>> hashMap) {
```

Comment:
what is the purpose of this method?

Reply:
Wait replication data reach at the standby cluster.